core/src/types/operator/metadata.rs

```diff
@@ -57,6 +57,11 @@ impl OperatorInfo {
         self.0.capabilities().contains(AccessorCapability::Write)
     }
 
+    /// Check if current backend supports [`Accessor::copy`] or not.
+    pub fn can_copy(&self) -> bool {
```

Comment:
This logic is wrong. Copy is different from read A and write B. We only implement Copy when backend has native copy support.

Reply:
How to define `native copy support`? 
Some backends don't provide copy support directly, but we can do so by combining several different operations and it won't be complex.

Reply:
> Some backends don't provide copy support directly, but we can do so by combining several different operations and it won't be complex.

Native support refers to a backend's ability to perform an operation using its native API without requiring additional workarounds or hacks. This type of support typically results in better performance. 

For instance, the `fs` service supports native copy by utilizing the syscall provided by different platforms such as `copy_file_range` on Linux. Similarly, the `s3` service supports native copy through the use of `CopyObject`.

It is essential for services to report their level of native support accurately so that operators can choose whether to utilize it or fallback to read and write logic.

In summary, all services should only provide native support and return 'NotSupported' for operations they do not support.

Reply:
> but we can do so by combining several different operations and it won't be complex.

Yes, we can do this. But not in service level.